crypto/rand/rand_lib.c

```diff
@@ -46,7 +46,7 @@ void rand_read_tsc(RAND_poll_fn cb, void *arg)
     unsigned char c;
     int i;
 
-    for (i = 0; i < 10; i++) {
+    for (i = 0; i < TSC_READ_COUNT; i++) {
         c = (unsigned char)(OPENSSL_rdtsc() & 0xFF);
```

Comment:
In SGX they bounce back out to real processor as i understand it :)
Thinking about bits and such hurts my brain and I cannot figure out what I should do -- I don't understand the hint.  Also, what should be added for the TSC check, since I can't find anything in the ia32cap.pod file.